### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by prankurpandeyy**
Things to watch : 

- UI 
- Header
- Footer 
- SIdebar
- Videocard

### CODE REVIEW

File: slate-video-library/src/Pages/Homepage/Homepage.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Footer from "../../Components/Footer/Footer";
+import Header from "../../Components/Header/Header";
+import Sidebar from "../../Components/Sidebar/Sidebar";
+import Videocard from "../../Components/Video-Card/Videocard";
```
Comment (aayushjain-code) [inline slate-video-library/src/Pages/Homepage/Homepage.js:5]: You can create a merge file for imports/exports and import every component from that file.
i.e.
`import  {Footer,Header,Sidebar ,VideoCard} from "../../Components"`

File: slate-video-library/src/Components/Video-Card/Videocard.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+.video__cards-title {
+  padding: 0.5rem;
+}
+.video__cards-details {
+  display: flex;
+  padding: 0.3rem;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(191, 191, 246);
```
Comment (aayushjain-code) [inline slate-video-library/src/Components/Video-Card/Videocard.css:38]: Try to avoid using different colors.
Use CSS Variables.

**Review by aayushjain-code — APPROVED** [review]
Try to focus more on structuring and naming as well Declaring and importing CSS Variables

**Review by JuvitaSaini09 — APPROVED** [review]
Looks nice

**Pull request merged**